src/components/Square.js

```diff
@@ -3,16 +3,20 @@ import PropTypes from 'prop-types';
 
 import './Square.css'
 
-const Square = (props) => {
+const Square = ({value, id, onClickCallback}) => {
   // For Wave 1 enable this 
-  //  Component to alert a parent 
-  //  component when it's clicked on.
+  const markSquare = () => {
+    return onClickCallback({
+      id: id,
+      value: value
+    })
+  }
 
-  return <button
-    className="square"
-  >
-    {props.value}
-  </button>
+  return (
+    <button onClick={() => {markSquare()}} className="square">
```

Comment:
I would suggest instead of making another function you could just call onClickCallBack() in the onClick attribute `onClick={() => onClickCallback(id)}`